cf-agent/package_module.c

```diff
@@ -1125,9 +1125,9 @@ PromiseResult FileInstallPackage(const char *package_file_path,
      
     if (action == cfa_warn || DONTDO)
     {
-         Log(LOG_LEVEL_WARNING, "Should install file type package: %s",
+         Log(LOG_LEVEL_VERBOSE, "Should install file type package: %s",
```

Comment:
So if I read this correctly, the reason this one and a few others are "verbose" and not "warning" is because of the `switch` statement in `verify_new_packages.c` that checks for `PROMISE_RESULT_WARN`, is that correct?